packages/backend-common/src/reading/FetchUrlReader.test.ts

```diff
@@ -77,6 +77,10 @@ describe('FetchUrlReader', () => {
               { host: 'example.com:700' },
               { host: '*.examples.org' },
               { host: '*.examples.org:700' },
+              {
+                host: 'foobar.org',
+                paths: ['/dir1/'],
+              },
```

Comment:
I guess this could have been implemented as a completely separate form,

```yaml
allow:
  host: host.com
  url: https://host.com/path/prefix
```

i think that was kind of the intent of the "host" key and additional nesting: to allow for future extension

Reply:
(we had a discussion internally and concluded that what you had here was better than my suggestion, so ignore this :P)